docs/api-reference/web-mercator-utils.md

```diff
@@ -48,9 +48,11 @@ Parameters:
 
 Returns:
 - `distanceScales` (Object)
-- `distanceScales.pixelsPerMeter` (Array) - pixels per meter in `[x, y, z]`
+- `distanceScales.pixelsPerMeter` (Array) - pixels per meter in `[x, y, z, x2]`
+    + `x2` adjusts `x` by y offset (in meters): `xAdjusted = x + x2 * dy)`. It offers a cheap way to compensate for the `x` changes with latitude.
 - `distanceScales.metersPerPixel` (Array) - meters per pixel in `[x, y, z]`
-- `distanceScales.pixelsPerDegree` (Array) - pixels per degree in `[x, y, z]`
+- `distanceScales.pixelsPerDegree` (Array) - pixels per degree in `[x, y, z, y2]`
```

Comment:
Probably a dumb question but why are we adjusting `x` for meters and `y` for degrees? We are using [lng, lat]  order instead of the often used (lat, lon) to match [x,y] orientation?